### PR TITLE
Add security md file

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

**Comment by outoftardis** [comment]
note that we have a [section about security in readme](https://github.com/oneapi-src/oneDAL#security-), if you are adding this file, it'd make sense to reference it from that section

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Report a Vulnerability
+
+Please report security issues or vulnerabilities to the [Intel® Security Center].
+
+For more information on how Intel® works to resolve security issues, see
```
Comment (outoftardis) [inline SECURITY.md:7]:
```suggestion
For more information on how Intel® resolves security issues, see
```

Comment (PetrovKP) [reply SECURITY.md:7]: I'm not sure if we can change the content here. @maria-Petrova It's true?

Comment (outoftardis) [reply SECURITY.md:7]: if it's a wording provided by legal, then it's ok to leave as it is

**Review by maria-Petrova — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (maria-Petrova) [reply SECURITY.md:7]: Yes, the wording was provided by legal, so please leave it the way it is.

**Pull request merged**